src/main/java/SaveAndLoad.java

```diff
@@ -0,0 +1,174 @@
+import java.io.*;
+import java.util.Arrays;
+
+public class SaveAndLoad {
+
+    enum TaskType {
+        ToDo, Deadline, Event
+    }
+
+    public static void save(Task[] listOfTasks) {
+        StringBuilder textOutput = new StringBuilder();
+        for (Task task : listOfTasks) {
+            if (task == null) {
+                break;
+            }
+            textOutput.append(task.toString()).append("\n");
+        }
+
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter("balkanbot.txt"));
+            writer.write(textOutput.toString());
+            writer.close();
+
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+    }
+
+    public static void markCheck(Task task, boolean completed) {
+        if (completed) {
+            task.mark();
+        }
+    }
+
+    public static void load(Task[] listOfTasks) {
```

Comment:
Maybe it be easier to save your logged values in a comma / hyphen separated format, the logged values could be the components needed for the constructor of your task. 

This would allow you to directly create the needed task using it's own constructor.

Additionally, this could also possibly make parsing the input easier too, as you can just call a .split(",") on a new line read by the buffered reader.

This could make this portion of the code lot clearer and reduce the code you have to manage :)